src/fhir-tool/Program.cs

```diff
@@ -895,14 +892,28 @@ private static Questionnaire.ItemComponent CreateItemComponentV1(QuestionnaireIt
             Questionnaire.ItemComponent itemComponent = new Questionnaire.ItemComponent
             {
                 Type = itemType,
-                LinkId = item.LinkId,
-                Prefix = string.IsNullOrEmpty(item.Prefix) ? null : item.Prefix,
-                Text = string.IsNullOrEmpty(item.Text) ? null : item.Text,
-                Required = item.Required.HasValue ? item.Required : null,
-                Repeats = item.Repeats,
-                ReadOnly = item.ReadOnly,
-                Initial = GetElement(itemType.Value, item.Initial)
             };
+            if (!string.IsNullOrEmpty(item.LinkId))
+                itemComponent.LinkId = item.LinkId;
+            if (!string.IsNullOrEmpty(item.Prefix))
+                itemComponent.Prefix = item.Prefix;
+            if (!string.IsNullOrEmpty(item.Text))
+                itemComponent.Text = item.Text;
+
+            if (itemType != Questionnaire.QuestionnaireItemType.Group && itemType != Questionnaire.QuestionnaireItemType.Display)
+            {
+                if (item.Required.HasValue)
+                    itemComponent.Required = item.Required;
+                itemComponent.ReadOnly = item.ReadOnly;
```

Comment:
no guard for this one as well. Intentional?

Reply:
Yep, reason same as above